Source/Core/DolphinQt2/QtUtils/ElidedButton.h

```diff
@@ -0,0 +1,20 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <QPushButton>
+
+class ElidedButton : public QPushButton
+{
+public:
+  ElidedButton(const QString& text = QStringLiteral(""),
+               Qt::TextElideMode elide_mode = Qt::ElideRight);
+  Qt::TextElideMode elideMode() const;
```

Comment:
I kept this in the Qt style to match other eliding QtWidgets, e.g. http://doc.qt.io/qt-4.8/qtabwidget.html#elideMode-prop